Workbooks/PostgreSQL Flexible Server/Performance/Autovacuum Blockers/Autovacuum Blockers.workbook

```diff
@@ -1206,7 +1206,7 @@
                 {
                   "type": 1,
                   "content": {
-                    "json": "- If the transaction duration is > {MIN_XACT_TIME_MS_THRESHOLD_FOR_LONG_XACTS} ms, we consider it as long-running transaction.\r\n- Prevents Autovacuum from reclaiming space in the affected tables\r\n- Can cause bloat in tables and indexes, leading to degraded performance over time\r\n- Best practice is to keep transactions as short as possible",
+                    "json": "- Transactions with a duration exceeding 7,200,000 ms are classified as long-running transactions.\r\n- Long-running transactions do hinder Autovacuum's ability to reclaim space in affected tables.\r\n- These transactions may contribute to table and index bloat, resulting in performance degradation over time.\r\n- To maintain optimal performance, it is recommended to keep transactions as brief as possible.",
```

Comment:
Can you not hardcode 7,200,000 ms. Please fallback to MIN_XACT_TIME_MS_THRESHOLD_FOR_LONG_XACTS

Reply:
![image](https://user-images.githubusercontent.com/127991103/236271145-e7557610-da5f-4569-946c-ecf71da55ba5.png)

Updated